patches/0022-content-Disable-the-WASM-trap-handler-on-Linux.patch

```diff
@@ -0,0 +1,26 @@
+From 6bfe197bcf785bd6aaec995b0c2ae605cc6a0a44 Mon Sep 17 00:00:00 2001
+From: Ryan Gonzalez <rymg19@gmail.com>
+Date: Wed, 19 May 2021 16:20:55 -0500
+Subject: [PATCH 22/22] content: Disable the WASM trap handler on Linux
```

Comment:
Why is this desirable?

Reply:
@wjt Slightly worse performance on bounds checks. I'd like to explore an alternative solution (I'd guess the sandbox is blocking the trap handler, but tbh I'm not really sure *why* yet), but this seems like a nasty enough bug (esp if it breaks *all* bounds checks, which I have unfortunately been unable to test) that adding a workaround until we know more is worthwhile.